Portfolio/index.css

```diff
@@ -1,174 +1,123 @@
-console.log('Hello World');
 
-body {
-    font-family: Arial, sans-serif;
-    margin: 0;
-    padding: 0;
-    background-color: #f0f0f0;

```

Comment:
The opening curly brace for the 'body' selector is missing at the beginning of line 7.